docs/pages/docs/apis/access-control.mdx

```diff
@@ -20,244 +20,171 @@ export default config({
-| `originalInput` | For `create` and `update` operations, this is the value of `data` passed into the mutation. For `read` and `delete` operations this value is `undefined`. |
-| `itemId`        | The `id` of the item being updated/deleted in `update` and `delete` operations. `undefined` for other operations.                                         |
-| `context`       | The [`KeystoneContext`](./context) object of the originating GraphQL operation.                                                                           |
-
-```typescript
-import { config, createSchema, list } from '@keystone-next/keystone';
+!> Filter access control cannot be used on `create` operations, as there is no pre-existing item to filter against.
 
-export default config({
-  lists: createSchema({
-    ListKey: list({
-      // Imperative access control definition
-      access: ({
-        listKey,
-        operation,
-        session,
-        originalInput,
-        itemId,
-        context,
-      }) => {
-        return true;
-      },
-    }),
-  }),
-});
-```
+### Item (mutations only)
 
-## Field Access Control
+`item` is the final and most powerful level of access control.
+It’s available to `create`, `update`, and `delete` mutations, and lets you write functions which have access to:
 
-Keystone also allows you to set up access control on a per-field basis.
-The default access control is to follow the access control rules for the parent list.
-Access control is applied to the generated CRU (**c**reate, **r**ead, **u**pdate, but not delete) queries and mutations in the [GraphQL API](./graphql).
-Field access control is applied after list access control has been applied.
+- the **input data** of the mutation (for `create` and `update` operations), and
+- the **existing item** in the database (for `update` and `delete` operations).
 
-You can specify access control using either **concise** or **verbose** syntax.
-Verbose syntax provides a separate access control rule for each operation type; `create`, `read`, and `update`.
-Concise syntax provides a single access control rule which is used for all operation types.
+!> These functions must return `true` if the operation is allowed, or `false` if it is not allowed.
 
 ```typescript
 import { config, createSchema, list } from '@keystone-next/keystone';
-import { text } from '@keystone-next/keystone/fields';
+import { checkbox } from '@keystone-next/keystone/fields';
 
 export default config({
   lists: createSchema({
     ListKey: list({
       fields: {
-        fieldName: text({
-          // Concise access control definition
-          access: true,
-          // Verbose access control definition
-          access: {
-            create: true,
-            read: true,
-            update: true,
-          },
-        }),
+        isReadable: checkbox(),
+        isUpdatable: checkbox(),
+        isDeletable: checkbox(),
```

Comment:
Since we're not using these fields in this example should we remove them for decluttering?

(or is this intended as a progressive build up from the last one...)

Reply:
Oh good catch, definitely should be removed!